config/options.yml

```diff
@@ -22,3 +22,46 @@ rpm:
   product_url:     https://github.com/ManageIQ/manageiq
 bundler_version:   2.1.4
 npm_registry:
+rpm_repository:
+  s3_api:
+    access_key:
+    secret_key:
+    bucket: rpm-manageiq-org
+    endpoint: nyc3.digitaloceanspaces.com
+  :arches:
+    - "src"
+    - "noarch"
+    - "x86_64"
+  :content:
+    :10-jansa:
+      :targets:
+        - el8
+      :rpms:
+        :libssh2:
```

Comment:
`libssh2` can be dropped as the CentOS repo now contains the package and we don't need to provide our own.